packages/flutter/lib/src/services/clipboard.dart

```diff
@@ -0,0 +1,35 @@
+// Copyright 2016 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+
+import 'package:sky_services/editing/editing.mojom.dart' as mojom;
+
+import 'shell.dart';
+
+export 'package:sky_services/editing/editing.mojom.dart' show ClipData;
+
+/// An interface to the system's clipboard. Wraps the mojo interface.
+class Clipboard {
+  Clipboard(this._service);
+  final mojom.Clipboard _service;
+
+  void setClipData(mojom.ClipData clip) {
+    _service.setClipData(clip);
+  }
+
+  Future<mojom.ClipData> getClipData() async {
+    mojom.ClipboardGetClipDataResponseParams params = await _service.getClipData();
+    return params.clip;
+  }
+}
+
+mojom.ClipboardProxy _initClipboardProxy() {
+  mojom.ClipboardProxy proxy = new mojom.ClipboardProxy.unbound();
+  shell.connectToService('mojo:clipboard', proxy);
+  return proxy;
+}
+
+final mojom.ClipboardProxy _clipboardProxy = _initClipboardProxy();
+final Clipboard clipboard = new Clipboard(_clipboardProxy.ptr);
```

Comment:
A better pattern to follow is SystemSound or SystemChrome.  We've been using static functions on a class rather than having a singleton instance.  That makes the calling code slightly clearer about what's going on because the singletone doesn't look like a stealth local variable.